pkg/cri/resource-manager/policy/builtin/balloons/balloons-policy.go

```diff
@@ -935,13 +942,14 @@ func (p *balloons) setConfig(bpoptions *BalloonsOptions) error {
 		return err
 	}
 	balloons = append(balloons, defaultBalloon)
+	p.balloons = balloons
```

Comment:
Very minor nit, do we need the `balloons` variable after this, could we get rid of it?

Reply:
Thanks @jukkar! Fixed.